src/components/SharedComponents/StyledRegionName/StyledRegionName.tsx

```diff
@@ -3,21 +3,27 @@
  * Used in Compare table + geolocated region links on the homepage.
  */
 import React, { Fragment } from 'react';
-import { getRegionNameForRow } from 'common/utils/compare';
 import { Region, getFormattedStateCode } from 'common/regions';
 import { Suffix, Wrapper } from './StyledRegionName.style';
+import { getSplitRegionName } from 'components/NewLocationPage/LocationName/utils';
 
 const StyledRegionName: React.FC<{
   showStateCode: boolean;
   region: Region;
   condensed?: boolean;
   truncateText?: boolean;
 }> = ({ showStateCode, region, condensed, truncateText = false }) => {
-  const [regionNameMain, regionSuffix] = getRegionNameForRow(region, condensed);
+  const [regionNameMain, regionSuffix] = getSplitRegionName(region, condensed);
 
   return (
     <Wrapper $truncateText={truncateText}>
-      {regionNameMain} {regionSuffix && <Suffix>{regionSuffix}</Suffix>}
+      {regionNameMain}
+      {regionSuffix && (
+        <Suffix>
+          {regionSuffix !== 'metro' ? ` ${regionSuffix}` : ` ${regionSuffix},`}
+        </Suffix>
+      )}
+      {!regionSuffix && region.regionType !== 'state' && <Suffix>,</Suffix>}
```

Comment:
when does this return true (not a state but doesnt have a suffix)?

Reply:
Added comments to clarify :)